src/SFA.DAS.AANHub.Database/Tables/MemberRegion.sql

```diff
@@ -0,0 +1,7 @@
+﻿CREATE TABLE [dbo].[MemberRegion]
+(
+	[MemberId] UNIQUEIDENTIFIER NOT NULL PRIMARY KEY, 
```

Comment:
Primary key should be composite of MemberId and RegionId

Reply:
Done